scapy/plist.py

```diff
@@ -85,6 +85,19 @@ def __repr__(self):
                                s,
                                ct.punct(">"))
 
+    def __getstate__(self):
```

Comment:
Could you add a doctring?

Reply:
Yeah, I was thinking about that... and than I forgot about that... And you are right.